account_auto_reconcile_rounding_differences/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
Here you should put the License like it's said in the manteiner-tools "init" template (https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py#L2)